BDArmory/Weapons/BDCustomWarhead.cs

```diff
@@ -8,6 +8,7 @@
 using BDArmory.Bullets;
 using static BDArmory.Bullets.PooledBullet;
 using static BDArmory.Weapons.ModuleWeapon;
+using static KSP.UI.Screens.Settings.SettingsSetup;
```

Comment:
Lots of unused `using` statements in here. VS has a tendency to include stuff whenever something auto-completes to the wrong thing if you don't immediately undo it.

Reply:
Yup, I've fixed it in other cases but it slipped through here. Fixed.